docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -4,9 +4,7 @@ lang: en
 ---
 
 ## Overview
-Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment.
-## Important Considerations
-We've capped usage for beta at 20 hours for a maximum of 30 days. We highly recommend that you save your work and close the browser tab that is running Code Builder to stop the usage clock when you aren’t using it.
+Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment. Before you start, see [Code Builder Overview](./en/codebuilder/about) for more information and important considerations.
```

Comment:
Do you want them to do the latter before they go to the Quick Start? Or are you just providing a list of "quick start" content? 
Before you begin:

- Complete to the Quick Start to yadda yadda
- Before you dive deeper, see Code Builder Overview for blah blah

Reply:
We just want them to read the overview and look at the important considerations section. We were sort of duplicating the text from the "Important Considerations" section here. I just consolidated it.